projects/vdk-plugins/vdk-kerberos-auth/src/vdk/plugin/kerberos/kerberos_plugin.py

```diff
@@ -56,6 +57,9 @@ def vdk_configure(config_builder: ConfigurationBuilder) -> None:
 
     @hookimpl
     def vdk_initialize(self, context: CoreContext) -> None:
+        disable_oauth = os.getenv("DISABLE_KERBEROS_LOGIN").lower() == "true"
```

Comment:
Please move the configuration to the KerberosPluginConfiguration class in kerberos_configuration.py